tests/dispatch_test.c

```diff
@@ -82,6 +84,35 @@ dispatch_test_check_evfilt_read_for_fd(int fd)
 #endif
 }
 
+const char *
+dispatch_test_get_test_file(void)
+{
+#if defined(_WIN32)
+	static char path[MAX_PATH];
+	DWORD len = GetModuleFileNameA(NULL, path, sizeof(path));
+	if (len == 0 || len == sizeof(path)) {
+		fprintf(stderr, "GetModuleFileNameA() failed: %u\n", GetLastError());
+		exit(EXIT_FAILURE);
+	}
+	return path;
```

Comment:
Uh, this is returning a buffer to a stack allocated buffer.  This is a classic use-after-free.

Reply:
The buffer is static.